<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="launch_btn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Launch</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="settings_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_edit">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>250</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>username</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ver_box">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>version</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="install_pb">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>420</y>
      <width>641</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
